staging/cse/windows/containerdfunc.ps1

```diff
@@ -182,12 +179,25 @@ function Install-Containerd {
     $hypervRuntimes = CreateHypervisorRuntimes -builds @($hypervHandlers) -image $pauseImage
   }
 
-  # remove the value containerAnnotations and podAnnotations place holder since it is not supported in containerd versions older than 1.7.9
-  if (([version]$containerdVersion).CompareTo([version]"1.7.9") -lt 0) {
-    # remove the value containerAnnotations place holder
-    $template = $template | Select-String -Pattern 'containerAnnotations' -NotMatch
-    # remove the value podAnnotations place holder
-    $template = $template | Select-String -Pattern 'podAnnotations' -NotMatch
+  try {
+    # remove the value containerAnnotations and podAnnotations place holder since it is not supported in containerd versions older than 1.7.9
+    pushd $global:ContainerdInstallLocation

```

Comment:
Approved. Just a question: Why do you use `pushd` instead of absolute path `$global:ContainerdInstallLocation\containerd.exe`?

Reply:
Tried `$global:ContainerdInstallLocation\containerd.exe` but it does not work

Reply:
At line:1 char:34
+ $global:ContainerdInstallLocation\containerd.exe -v
+                                  ~~~~~~~~~~~~~~~
Unexpected token '\containerd.exe' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken

Reply:
'C:\Program Files\containerd\containerd.exe' -v
At line:1 char:46
+ 'C:\Program Files\containerd\containerd.exe' -v
+                                              ~~
Unexpected token '-v' in expression or statement.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : UnexpectedToken

Reply:
```
& 'C:\Program Files\containerd\containerd.exe' -v
containerd github.com/containerd/containerd v1.7.9+azure 4f03e100cb967922bec7459a78d16ccbac9bb81d
```
It seems like that adding `&` before the path works. Keep the current code since there is no issue.